hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/exports/recordstreaming/RecordStreamingUtils.java

```diff
@@ -164,8 +164,8 @@ private static boolean isRelevant(final String file) {
         return file.endsWith(V6_FILE_EXT) || file.endsWith(V6_GZ_FILE_EXT);
     }
 
-    public static int compareSidecarFiles(final String a, final String b) {
-        return SIDECAR_FILE_COMPARATOR.compare(a, b);
+    public static int compareSidecarFiles(final String lhs, final String rhs) {
```

Comment:
This seems excessive

Reply:
This follows our adopted Google Code Style standard: https://google.github.io/styleguide/javaguide.html#s5.2.6-parameter-names